content/blogposts/2024/announcing-prompt-library.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Announcing Prompt Library"
+authors:
+  - name: YK Sugi
+    url: https://x.com/ykdojo
+publishDate: 2024-11-25T10:00-01:00
+description: "Introducing Prompt Library: Save, reuse, and share frequently used prompts for Cody"
+tags: [blog]
+slug: 'announcing-prompt-library'
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/towards-infinite-context-for-code/infinite-context-og.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/towards-infinite-context-for-code/infinite-context-og.png
+---
+
+As you work with Sourcegraph’s Cody, you might find yourself repeating the same prompt over and over again. And you might want to share it with your coworkers or friends.
+
+Introducing **Prompt Library**: a way to save and reuse frequently used prompts, as well as share them within your organization or publicly.
+
+Let’s take a look at how it works:
+
+1.	Go to Sourcegraph’s web interface.
+
+2.	From the top menu, find **Tools** and then **Prompt Library**.
+
+![image.png](https://sourcegraph.notion.site/image/https%3A%2F%2Fprod-files-secure.s3.us-west-2.amazonaws.com%2Fe7ce844a-fe2e-4102-b77e-e852aee3841b%2F9a4e60f9-d0d6-4555-8dec-1c1e666d6985%2Fimage.png?table=block&id=142a8e11-2658-8080-b8bb-e21b5d503124&spaceId=e7ce844a-fe2e-4102-b77e-e852aee3841b&width=2000&userId=&cache=v2)
+
+3.	Click **Create new prompt**, set the prompt name, description, and the prompt itself, then click **Create prompt**.
+
+![image.png](https://sourcegraph.notion.site/image/https%3A%2F%2Fprod-files-secure.s3.us-west-2.amazonaws.com%2Fe7ce844a-fe2e-4102-b77e-e852aee3841b%2Fb793b037-8544-49a5-beae-5db4b070fff0%2Fimage.png?table=block&id=142a8e11-2658-8085-9d3f-d2abd19566be&spaceId=e7ce844a-fe2e-4102-b77e-e852aee3841b&width=2000&userId=&cache=v2)
+
+![image.png](https://sourcegraph.notion.site/image/https%3A%2F%2Fprod-files-secure.s3.us-west-2.amazonaws.com%2Fe7ce844a-fe2e-4102-b77e-e852aee3841b%2F10c88996-bb30-4aca-b646-167003735728%2Fimage.png?table=block&id=142a8e11-2658-8014-a2a1-cae0d8dc95cd&spaceId=e7ce844a-fe2e-4102-b77e-e852aee3841b&width=2000&userId=&cache=v2)
+
+**Once created:**
+
+You can go to Cody Chat, select your saved prompts, add any necessary details, and run them.
+
+![image.png](https://sourcegraph.notion.site/image/https%3A%2F%2Fprod-files-secure.s3.us-west-2.amazonaws.com%2Fe7ce844a-fe2e-4102-b77e-e852aee3841b%2Fc57ee65b-31cf-40de-841d-c53ceca0fa0d%2Fimage.png?table=block&id=142a8e11-2658-8099-8cf5-d613adaea23b&spaceId=e7ce844a-fe2e-4102-b77e-e852aee3841b&width=2000&userId=&cache=v2)
```

Comment:
Can we swap this one out for a gif showing how to use it?

Reply:
Sure, will do